src/runtime/cuda/nccl_api.cu

```diff
@@ -178,7 +178,7 @@ std::pair<IdArray, NDArray> SparsePush(
   // permute the indices and values
   if (num_in > 0) {
     const dim3 block(256);
-    const dim3 grid((num_in+block.x-1)/block.x);
+    const dim3 grid((num_in+block.x-1)/block.x == 0 ? 1 : (num_in+block.x-1)/block.x);
```

Comment:
Now that the `if (num_in > 0) {` was added above, this change is no longer needed.

Reply:
I c, will remove it.